compare_runs.py

```diff
@@ -0,0 +1,56 @@
+import filecmp
+import getopt
+import os
+import sys
+import subprocess
+
+
+def get_arguments(argv):
+    bc_file = ''
+    output_bc = ''
+    test_count = 2
+    flags = ''
+    logfile_name = 'logfile'
+    try:
+        opts, args = getopt.getopt(argv,"hi:o:n:f:l:",[])
+    except getopt.GetoptError:
+        print('compare_runs.py -i <input_bc_file> -o <output_bc_file> -n <re-run_counts> -f "<clang_run_flags>" -l "<log_file_name>"')
+        sys.exit(2)
+    print(opts)
+    for opt, arg in opts:
+        if opt == '-h':
+            print('compare_runs.py -i <input_bc_file> -o <output_bc_file> -n <re-run_counts> -f "<clang_run_flags>" -l "<log_file_name>"')
```

Comment:
I have tried some examples with "-" in the flags and it works. I don't have a good example with "=" to test it with but if you have something in mind, I can try but I think since that part is in quotes it shouldn't have any issues.

Reply:
I tried  -DFEATURE=1 and it works.